spec/services/zendesk/pvb_api_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.describe Zendesk::PVBApi do
+  let(:zendesk_api_client) { double(ZendeskAPI::Client) }
+  let(:zendesk_pvb_client) { Zendesk::PVBClient.instance }
+
+  subject { described_class.new(zendesk_pvb_client) }
+
+  before do
+    allow(zendesk_pvb_client).to receive(:request).and_yield(zendesk_api_client)
+  end
+
+  describe '#cleanup_tickets' do
+    let(:ticket_ids) { [{ id: 1 }, { id: 2 }, { id: 3 }].map { |t| ZendeskAPI::Ticket.new(zendesk_api_client, t) } }
+    let(:empty_ticket_ids) { [] }
+
+    let(:tickets) { ZendeskAPI::Collection.new(zendesk_api_client, ZendeskAPI::Ticket, ids: [1, 2, 3]) }
+    let(:empty_tickets) { ZendeskAPI::Collection.new(zendesk_api_client, ZendeskAPI::Ticket, ids: []) }
+
+    let(:twelve_months_ago) { 12.months.ago.strftime('%Y-%m-%d') }
+    let(:query) do
+      {
+        query: "type:ticket tags:staff.prison.visits updated<#{twelve_months_ago}",
+        reload: true
+      }
+    end
+
+    before do
+      expect(zendesk_api_client).to receive(:search).
+        and_return(ticket_ids, empty_ticket_ids)
+      expect(zendesk_api_client).to receive(:tickets).and_return(tickets)
+    end
+
+    it 'deletes tickets that have not been updated in twelve months or less' do
+      allow(tickets).to receive(:destroy_many!).and_return(tickets)
```

Comment:
I think you can get rid of that line and append the `and_return(tickets)` to lines 38..40